lab-andrew/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This is good, you'll need to move the 'del' module into dependencies if you continue using it since your server cannot run without it.  It doesn't get used by the storage module though so you may be able to remove it entirely.